vllm/v1/serial_utils.py

```diff
@@ -158,6 +158,8 @@ def _encode_tensor(
         self, obj: torch.Tensor
     ) -> tuple[str, tuple[int, ...], Union[int, memoryview]]:
         assert self.aux_buffers is not None
+        # this creates a copy of the tensor if it's not already contiguous
+        obj = obj.contiguous()
```

Comment:
@lgeiger might be better to do this after flattening? (I know it wasn't that order originally but I think it's logically better in that order).

Reply:
Good point, that's much clearer. Done in 1bc3bf38a8427b4efbdcce35272b94b51b145480